### DISCUSSION TITLE
Welcome to CIS-Ubuntu-20.04-Ansible Discussions!

### DISCUSSION THREAD

**Discussion by alivx** [alivx/CIS-Ubuntu-20.04-Ansible#13] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gitcoleman** [comment]
Hi,

I'm very new to ansible and i'm trying t harden my Ubuntu 20 server using CIS-Ubuntu-20.04-Ansible, however, i am unable to run the playbook.  i've cloned the githup project to my server and i've changed everything in the defaults/main.yml that i need to change.  I'm trying to run the playbook with this command, "ansible-playbook defaults/main.yml" but then i get this error, "

"root@ip-172-31-34-60:/home/ubuntu/CIS_Hardening/CIS-Ubuntu-20.04-Ansible# ansible-playbook defaults/main.yml
[WARNING]:  * Failed to parse /etc/ansible/hosts with yaml plugin: YAML inventory has invalid structure, it should be a dictionary, got: <class 'ansible.parsing.yaml.objects.AnsibleSequence'>
[WARNING]:  * Failed to parse /etc/ansible/hosts with ini plugin: /etc/ansible/hosts:44: Expected key=value host variable assignment, got: hosts:
[WARNING]: Unable to parse /etc/ansible/hosts as an inventory source
[WARNING]: No inventory was parsed, only implicit localhost is available
[WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'
ERROR! A playbook must be a list of plays, got a <class 'ansible.parsing.yaml.objects.AnsibleMapping'> instead

The error appears to be in '/home/ubuntu/CIS_Hardening/CIS-Ubuntu-20.04-Ansible/defaults/main.yml': line 5, column 1, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

# Section 1 settings
disable_cramfs: yes
^ here"

please let me know what i am doing wrong.  Also, i need to run this locally.

**Reply by alivx** [reply]
You need to set ` -i inventory` 
Under inventory file
set server ip

**Reply by gitcoleman** [reply]
Thanks, got it working.

**Comment by datechnic** [comment]
Any plans for scripting v2.1.1 CIS benchmarks for Ubuntu 20.04 ?

**Comment by berthou** [comment]
Hello @alivx !
Thank you so much for the hard work ! This role is pretty useful !
I'm using it to play around and learn about server hardening.
Cheers,
berthou
